docs/installation/post-install.rst

```diff
@@ -26,12 +26,12 @@ path for the partitions to use:
    apiVersion: storage.metalk8s.scality.com/v1alpha1
    kind: Volume
    metadata:
-     name: <node_name>-prometheus
+     name: bootstrap-prometheus
```

Comment:
No, this would only be the case when deploying Prometheus and AlertManager on the bootstrap node, which we explicitly recommend against above.